resources/dicts/thoughts/dead/darkforest/newborn.json

```diff
@@ -12,7 +12,13 @@
             "Hears menacing whispers around {PRONOUN/m_c/object}",
             "Is clouded in confusion and fear",
             "Feels trapped in the sticky mud",
-            "Is feeling incredibly lonely"
+            "Is feeling incredibly lonely",
+            "Feels intense hatred for this fate",
+            "Is happy {PRONOUN/m_c/subject} can't see the scary place around {PRONOUN/m_c/object}",
+            "Is starting to feel comfortable in this dreary place",
+            "Endlessly hungry",
```

Comment:
youll want to add somehting like "is" to the start of this. remember that they need to read like a sentence that starts with the cat's name